Frequently-Asked-Questions.md

```diff
@@ -59,3 +59,15 @@ Orleans currently does not support updating grain code on the fly. To upgrade to
 ## Can I persist a grain’s state to the Azure cache service?
 
 This can be done though a storage provider for Azure Cache. We don’t have one but you can easily build your own.
+
+## Can I Connect to Orleans silos from the public internet?
+
+Orleans is Designed to be hosted as the back-end part of a service and you are suposed to create a front-end in your servers which clients connect to. It can be a http based Web API project, a socket server, a signalR server or anything else which you require.
```

Comment:
Low case "designed". Upper case "SignalR". Maybe it's worth adding (start with that?) that it is technically possible for clients to connect to silos over public internet, it is strongly not recommended because for security reasons.